testsuite/MDAnalysisTests/lib/test_util.py

```diff
@@ -21,37 +21,39 @@
 #
 from __future__ import absolute_import, division
 
-from unittest import TestCase
-
-from six.moves import range, StringIO
-import six
-
-import numpy as np
-from numpy.testing import (assert_raises, assert_equal, assert_almost_equal,
-                           assert_array_almost_equal, assert_, assert_array_equal)
-
 import MDAnalysis as mda
-import MDAnalysis.lib.util as util
 import MDAnalysis.lib.mdamath as mdamath
-from MDAnalysis.lib.util import cached
+import MDAnalysis.lib.util as util
+import numpy as np
+import pytest
+import six
 from MDAnalysis.core.topologyattrs import Bonds
 from MDAnalysis.exceptions import NoDataError
-
-
+from MDAnalysis.lib.util import cached
 from MDAnalysisTests.datafiles import Make_Whole
+from numpy.testing import (assert_equal, assert_almost_equal,
+                           assert_array_almost_equal, assert_array_equal)
+from six.moves import range, StringIO
 
 
-def check_parse_residue(rstring, residue):
-    assert_equal(util.parse_residue(rstring), residue)
-
-
-def check_convert_aa_3to1(resname3, resname1):
-    assert_equal(util.convert_aa_code(resname3), resname1)
-
-
-def check_convert_aa_1to3(resname1, resname3_canonical):
-    assert_equal(util.convert_aa_code(resname1), resname3_canonical)
+def test_convert_aa_code_long_data():
```

Comment:
Is this getting also ran as a test because of the name? 

Could this get rewritten as a generator?